configure.ac

```diff
@@ -0,0 +1,191 @@
+        [ORG_PATH="github.com/coreos"])
+
+AC_SUBST(ORG_PATH)
+
+# STAGE1 build settings
+
+## STAGE1:type
+AC_ARG_WITH(stage1,
+        AS_HELP_STRING([--with-stage1=type],
+                [type of stage1 build one of 'src', 'coreos', 'none']),
+        [RKT_STAGE1_USR_FROM="$withval"],
+        [RKT_STAGE1_USR_FROM="coreos"])
+
+case "${RKT_STAGE1_USR_FROM}" in
+	none)
+        AC_MSG_RESULT([stage1 build disabled])
+		;;
+	coreos)
+        AC_MSG_RESULT([stage1 will reuse coreos image])
+        
+        # check that some coreos-build specific binaries are here
+        AC_CHECK_PROG([CURL],[curl],[curl])
+        if test x"${CURL}" == x ; then
+            AC_MSG_ERROR([curl not found])
+        fi
+
+        AC_CHECK_PROG([GPG],[gpg],[gpg])
+        if test x"${GPG}" == x ; then
+            AC_MSG_ERROR([gpg not found])
+        fi
+
+        AC_CHECK_PROG([MKTEMP],[mktemp],[mktemp])
+        if test x"${MKTEMP}" == x ; then
+            AC_MSG_ERROR([mktemp not found])
+        fi
+
+        AC_CHECK_PROG([MD5SUM],[md5sum],[md5sum])
+        if test x"${MD5SUM}" == x ; then
+            AC_MSG_ERROR([md5sum not found])
+        fi
+
+        AC_CHECK_PROG([CPIO],[cpio],[cpio])
+        if test x"${CPIO}" == x ; then
+            AC_MSG_ERROR([cpio not found])
+        fi
+
+        AC_CHECK_PROG([GZIP],[gzip],[gzip])
+        if test x"${GZIP}" == x ; then
+            AC_MSG_ERROR([gzip not found])
+        fi
+
+        AC_CHECK_PROG([UNSQUASHFS],[unsquashfs],[unsquashfs])
+        if test x"${UNSQUASHFS}" == x ; then
+            AC_MSG_ERROR([unsquashfs not found])
+        fi
+
+        AC_CHECK_PROG([SORT],[sort],[sort])
+        if test x"${SORT}" == x ; then
+            AC_MSG_ERROR([sort not found])
+        fi
+        
+		;;
+	src)
+        AC_MSG_RESULT([stage1 will build systemd from source])
+
+        # these are needed for systemd build
+        AC_CHECK_PROG([INTLTOOLIZE],[intltoolize],[intltoolize])
+        if test x"${INTLTOOLIZE}" == x ; then
+            AC_MSG_ERROR([intltoolize not installed, apt-get install intltool])
+        fi
+
+        AC_CHECK_PROG([LIBTOOLIZE],[libtoolize],[libtoolize])
+        if test x"${LIBTOOLIZE}" == x ; then
+            AC_MSG_ERROR([libtoolize not installed, apt-get install libtool])
+        fi
+		;;
+	*)
+		AC_MSG_ERROR([stage1 build type '${RKT_STAGE1_USR_FROM}' unsupported])
+esac
+
+AC_SUBST(RKT_STAGE1_USR_FROM)
+
+## STAGE1: Systemd git path
+AC_ARG_WITH(stage1-systemd-src,
+        AS_HELP_STRING([--with-stage1-systemd-src=git-path],
+                [address to git repository of systemd, used in 'src' build mode]),
+        [RKT_STAGE1_SYSTEMD_SRC="$withval"],
+        [RKT_STAGE1_SYSTEMD_SRC="git://anongit.freedesktop.org/systemd/systemd"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_SRC)
+
+## STAGE1: Systemd version
+AC_ARG_WITH(stage1-systemd-version,
+        AS_HELP_STRING([--with-stage1-systemd-version=version],
+                [systemd version to build]),
```

Comment:
Please add an information about default value in help string (see above for an example).